quality_control/models/qc_trigger_line.py

```diff
@@ -14,8 +14,12 @@ class QcTriggerLine(models.AbstractModel):
     user = fields.Many2one(
         comodel_name='res.users', string='Responsible',
         track_visibility='always', default=lambda self: self.env.user)
+    partner = fields.Many2many(
```

Comment:
OK, seems a mistake when replacing some fields... Travis fails because this change...